src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -2862,17 +2898,28 @@ def calculate_NPV(b, fixed_OM, variable_OM):
         example, if the capital expenditures are distributed across a 3-year
         capital escalation period, the PV from the capital costs are given as
 
-        PV_Capital_Cost = Y1_% * CAPEX * P/A(iLoan_%, gCap, 1)
-                          + Y2_% * CAPEX * P/A(iLoan_%, gCap, 2)
-                          + Y3_% * CAPEX * P/A(iLoan_%, gCap, 3)
+        PV_Capital_Cost = Y1_% * CAPEX * [P/A(r, gCap, 1) - P/A(r, gCap, 0)]
+                          + Y2_% * CAPEX * [P/A(r, gCap, 2) - P/A(r, gCap, 1)]
+                          + Y3_% * CAPEX * [P/A(r, gCap, 3) - P/A(r, gCap, 2)]
 
         where Y1_%, Y2_%, and Y3_% are the percentages of capital expenditure
         in each year expressed as decimals, CAPEX is the total capital cost from
         equipment purchasing, gCap is the capital escalation growth rate
-        expressed as a decimal, and iLoan_% is the capital equipment loan
-        interest rate expressed as a decimal. The capital costs spent in each
-        year are handled separately to properly account for the value growth
-        over time.
+        expressed as a decimal. The capital costs spent in each year are handled
+        separately to properly account for the value growth over time. Loan
+        repayment and interest owed are calculated as
+
+        Annual_Loan_Payment = Debt * A/P(iLoan_%, 0, Nloan) = Debt / P/A(iLoan_%, 0, Nloan)
```

Comment:
Would it make sense to combine the annual loan payment and interest into a single term for simplicity?

Reply:
Since we're using the series present worth factor now, I agree that it would make sense to combine and simplify the expressions.